# DP: This patch was converted from alpha-ieee-doc.diff
# DP: Original comment:
# DP:
# DP: #212912
# DP: on alpha-linux, make -mieee default and add -mieee-disable switch
# DP: to turn default off (doc patch)

Index: b/gcc/doc/invoke.texi
===================================================================
--- a/gcc/doc/invoke.texi	2012-04-03 20:37:53.000000000 -0400
+++ b/gcc/doc/invoke.texi	2012-04-03 20:37:57.000000000 -0400
@@ -12035,6 +12035,13 @@
 values such as not-a-number and plus/minus infinity.  Other Alpha
 compilers call this option @option{-ieee_with_no_inexact}.
 
+DEBIAN SPECIFIC: This option is on by default, unless
+@option{-ffinite-math-only} (which is part of the @option{-ffast-math}
+set) is specified, because the software functions in the GNU libc math
+libraries generate denormalized numbers, NaNs, and infs (all of which
+will cause a programs to SIGFPE when it attempts to use the results without
+@option{-mieee}).
+
 @item -mieee-with-inexact
 @opindex mieee-with-inexact
 This is like @option{-mieee} except the generated code also maintains
